devices/lixee.js

```diff
@@ -0,0 +1,475 @@
+    ],
+    configure: async (device, coordinatorEndpoint, logger) => {
+        const SW1_ENDPOINT = 1;
+        const endpoint = device.getEndpoint(SW1_ENDPOINT);
+
+        await reporting.bind(endpoint, coordinatorEndpoint, [
+            clustersDef._0x0702, /* seMetering */
+            clustersDef._0x0B01, /* haMeterIdentification */
+            clustersDef._0x0B04, /* haElectricalMeasurement */
+            clustersDef._0xFF66, /* liXeePrivate */
+        ])
+            .catch((e) => {
+                logger.error(`ZLINKY_DEBUG: failed to bind: ${e.message}`);
+            });
+
+        // ZLinky don't emit divisor and multiplier
+        await endpoint.saveClusterAttributeKeyValue('haElectricalMeasurement', {acCurrentDivisor: 1, acCurrentMultiplier: 1});
+        await endpoint.saveClusterAttributeKeyValue('seMetering', {divisor: 1, multiplier: 1});
+    },
+    onEvent: (type, data, device, options) => {
+        const endpoint = device.getEndpoint(1);
+        if (type === 'stop') {
+            clearInterval(globalStore.getValue(device, 'interval'));
+            globalStore.clearValue(device, 'interval');
+        } else if (type === 'start') {
+            getCurrentConfig(device, options)
+                .filter((e) => e.reportable) // we must subscribe just to reportable entities
+                .forEach((e) => {
+                    let change = 1;
+                    if (e.hasOwnProperty('reportChange')) {
+                        change = e['reportChange'];
+                    }
+
+                    endpoint
+                        .configureReporting(e.cluster, reporting.payload(e.exposes.property, 0, repInterval.HOUR, change))
+                        .catch((err) => {
+                            console.debug(`ZLINKY_DEBUG: Error ${e.cluster}/${e.exposes.property}: ${err}`);
+                            console.warn(`Error configuring report for ${e.cluster}/${e.exposes.property}`);
+                        });
+                });
+        } else if (!globalStore.hasValue(device, 'interval')) {
+            const seconds = options && options.measurement_poll_interval ? options.measurement_poll_interval : 60;
+
+            const interval = setInterval(async () => {
+                getCurrentConfig(device, options)
+                    .filter((e) => !e.reportable) // Just what must be readed manually
+                    .forEach((e) => {
+                        endpoint
+                            .read(e.cluster, [e.exposes.property])
+                            .catch((e) => console.debug(`ZLINKY_DEBUG: READ error ${e.cluster}/${e.exposes.property}: ${e}`));
+                    });
+            }, seconds * 1000);
+            globalStore.putValue(device, 'interval', interval);
+        }
+    },
+    endpoint: (dev) => {
+        return {
+            'haElectricalMeasurement': 1, 'liXeePrivate': 1, 'seMetering': 1, 'haMeterIdentification': 1, 'default': 1,
```

Comment:
I don't understand why this is needed, could you explain this?

Reply:
Honestly, I don't remember where I saw it. I will remove it